src/distr/bernoulli.rs

```diff
@@ -136,6 +136,16 @@ impl Bernoulli {
         let p_int = ((f64::from(numerator) / f64::from(denominator)) * SCALE) as u64;
         Ok(Bernoulli { p_int })
     }
+
+    #[inline]
+    /// Returns the probability (`p`) of the distribution.
```

Comment:
This should note that the output is a reconstruction which may differ from the input to `Bernoulli::new`.